src/vm/unixlowmem.cpp

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ===========================================================================
+
+#include "common.h"
+#include "unixlowmem.h"
+#include <stdlib.h>
+#include "debugmacros.h"
+
+#ifdef FEATURE_PAL
+
+UnixLowMemoryDetector::UnixLowMemoryDetector()
+{
+    LIMITED_METHOD_CONTRACT;
+
+    auto percent = max(min(ReadLowMemoryLimitPercent(), 100), 1);
+    this->m_szLowMemoryLimitBytes = ReadPhysicalMemoryLimitBytes() * percent / 100;
+}
+
+size_t UnixLowMemoryDetector::ReadLowMemoryLimitPercent()
+{
+    LIMITED_METHOD_CONTRACT;
+
+    char* defaultStackSizeStr = getenv("COMPlus_UnixLowMemoryLimitPercent");
```

Comment:
Could you please change it to use the standard config mechanism instead? That means adding an entry something like: 
```
RETAIL_CONFIG_DWORD_INFO(EXTERNAL_UnixLowMemoryLimitPercent, W"UnixLowMemoryLimitPercent", (DWORD)75, "Trigger GC if the process consumed more than the specified percentage of physical memory")
```
 to the src/inc/clrconfigvalues.h and then read it here using CLRConfig::GetConfigValue(CLRConfig::EXTERNAL_UnixLowMemoryLimitPercent).

In fact you can get rid of this method and just call the CLRConfig::GetConfigValue

Reply:
Important note - the values passed to all COMPlus_ variables are parsed as hex, so `export COMPlus_UnixLowMemoryLimitPercent=4B` would set it to those 75 percent.